Makefile error when `root_uri` is an S3 path

## 🐛 Bug
When running training command and having `root_uri` set to an S3 folder, this error shows up:
```
make: s3://<random_bucket_name>/predictions/Makefile: No such file or directory
make: *** No rule to make target 's3://<random_bucket_name>/predictions/Makefile'.  Stop.
```

This error disappears when `root_uri` is a local path. AWS config is right as it is able to read and write the files.

## To Reproduce

Steps to reproduce the behavior:

1. I ran the following command inside the container:
`python -m rastervision.pipeline.cli run local code/local_exp.py -a raw_uri s3://<random_bucket_name>/datafortesting/data/ -a root_uri s3://<random_bucket_name>/predictions -a test False`

<!-- Please provide the command executed, source of the get_config() function, error messages, and/or full stack traces if at all possible -->

## Expected behavior

It should run normally like it is running when `root_uri` is a local path.

## Environment

Running with docker.  **Image**: quay.io/azavea/raster-vision:pytorch-v0.13.1

## Additional context

This might be a relevant issue: #991

--- a/rastervision_pipeline/rastervision/pipeline/runner/local_runner.py
+++ b/rastervision_pipeline/rastervision/pipeline/runner/local_runner.py
@@ -2,7 +2,7 @@ import sys
 from os.path import dirname, join
 from subprocess import Popen
 
-from rastervision.pipeline.file_system import str_to_file
+from rastervision.pipeline.file_system import str_to_file, download_if_needed
 from rastervision.pipeline.runner.runner import Runner
 from rastervision.pipeline.utils import terminate_at_exit
 
@@ -68,7 +68,8 @@ class LocalRunner(Runner):
 
         makefile_path = join(dirname(cfg_json_uri), 'Makefile')
         str_to_file(makefile, makefile_path)
-        process = Popen(['make', '-j', '-f', makefile_path])
+        makefile_path_local = download_if_needed(makefile_path)
+        process = Popen(['make', '-j', '-f', makefile_path_local])
         terminate_at_exit(process)
         exitcode = process.wait()
         if exitcode != 0:

